ci/kubo-deployment.yml

```diff
@@ -123,12 +123,30 @@ resources:
     url: {{slack-url}}
 
 jobs:
+- name: run-unit-tests
+  plan:
+  - get: git-kubo-deployment
+  - get: git-kubo-release
+    version: every
+    trigger: true
```

Comment:
That's a great point. The intent here is we do not need to trigger the unit tests because they all live in kubo-release. However- that's not entirely true because the task itself (`run-unit-tests.yml`) is in git-kubo-deployment so we should still run it.

Without that additional trigger we'll still run the pipeline for kubo-deployment changes but we won't exercise the task if it changed.